src/applications/static-pages/i18Select/I18Select.js

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from 'react';
+import recordEvent from 'platform/monitoring/record-event';
+// TODO: feature toggle, styling
+const i18Content = {
+  en: {
+    label: 'English',
+    suffix: '/',
+  },
+  es: {
+    onThisPage: 'En esta página',
+    label: 'Español',
+    suffix: '-esp/',
+  },
+  tag: {
+    suffix: '-tag/',
+    label: 'Tagalog',
+    onThisPage: 'Tagalog On this page',
+  },
+};
+const langsToLinkSuffixes = {
+  es: '-esp/',
+  tag: '-tag/',
+};
+const I18Select = ({ baseUrls }) => {
+  const [lang, setLang] = useState('en');
+  useEffect(() => {
+    const contentDiv = document?.getElementById('content');
+
+    setLang('en');
+    if (contentDiv) {
+      contentDiv.setAttribute('lang', 'en');
+    }
+    for (const [key, value] of Object.entries(langsToLinkSuffixes)) {
+      if (document?.location.href.endsWith(value)) {
+        setLang(key);
+        if (contentDiv) {
+          contentDiv.setAttribute('lang', key);
+        }
+      }
+    }
+  }, []);
+
+  useEffect(
+    () => {
+      if (lang && lang !== 'en') {
+        const onThisPageEl = document?.getElementById('on-this-page');
+        onThisPageEl.innerText = i18Content[lang].onThisPage;
+      }
+    },
+    [lang],
+  );
+
+  const handleLinkClick = keyValue => {
+    recordEvent({
+      event: 'nav-covid-link-click',
+      faqText: undefined,
+      faqSection: undefined,
+    });
+
+    document.location.pathname = baseUrls[keyValue];
```

Comment:
@ncksllvn for context, this is where that object with urls is being used. to route to the associated language on link click. i figured this was a cleaner solution than a regex, but lmk if anything comes to mind